### PR TITLE
[new] Added prettycron@0.10.0

### PR DESCRIPTION
**Pull request by MMore**
See https://github.com/azza-bazoo/prettycron. `prettycron.js` minified using Uglifier to `prettycron.min.js`.

### CODE REVIEW

File: ajax/libs/prettycron/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "https://github.com/azza-bazoo/prettycron.git"
+  },
+  "author": "Hourann Bosci (http://hourann.com)",
+  "license": "LGPL",
+  "npmName": "prettycron",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/prettycron/package.json]: What's the reason why to include them all?
We don't even need `LICENSE.md`, `node_modules/`, `package.json`, `README.md` and `test/`.

**Comment by Piicksarn** [comment]
ping @cdnjs/intern @cdnjs/intern2

File: ajax/libs/prettycron/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/azza-bazoo/prettycron.git"
+  },
+  "author": "Hourann Bosci (http://hourann.com)",
+  "license": "LGPL",
+  "npmName": "prettycron",
+  "npmFileMap": [
+    {
+      "basePath": "/",
```
Comment (pvnr0082t) [inline ajax/libs/prettycron/package.json]: @MMore Let's use `""` as basePath.

File: ajax/libs/prettycron/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "prettycron",
+  "version": "0.10.0",
+  "description": "Display cron runspecs in human-readable format",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/azza-bazoo/prettycron.git"
+  },
+  "author": "Hourann Bosci (http://hourann.com)",
+  "license": "LGPL",
```
Comment (pvnr0082t) [inline ajax/libs/prettycron/package.json]: We should use `"LGPL-3.0"` here. It's more precise.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Pull request closed without merging**